src/components/profile/settings/contact-modal.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+    <div>
+        <button type="button" class="btn btn-outline-primary align-self-center" @click="showModal">
+            <slot></slot>
+        </button>
+        <client-only>
+            <Teleport to="#modal-container" key="contact-info-modal">
+                <div
+                    v-show="open"
+                    :class="{ 'modal-box': true, 'd-block': open, 'd-none': !open }"
+                    id="contact-info-modal"
+                >
+                    <div class="modal-content d-flex align-items-end flex-column">
+                        <div class="modal-header-box d-grid w-100">
+                            <div class="modal-heading sidenav-text my-auto mx-0 align-self-center">
+                                contact
+                            </div>
+                            <div class="modal-close-btn m-auto d-block" @click="closeModal">
+                                <PhX size="22" color="white"/>
+                            </div>
```

Comment:
Use bootstrap button

replace
```xml
                            <div class="modal-close-btn m-auto d-block" @click="closeModal">
                                <PhX size="22" color="white"/>
                            </div>
 ```
 with
```xml
                            <button class="btn btn-icon text-white fs-3" @click="closeModal">
                                <PhX />
                            </button>
```

Reply:
Worked on this.